src/components/Auth/PinSetup/PinPad/Header/styles.js

```diff
@@ -1,4 +1,6 @@
 import { StyleSheet } from 'react-native';
+import styled from 'styled-components';
```

Comment:
It's okay to use `styled-components` here, but it's not a good idea to mix `StyleSheet.create()` with `styled-components` together in the same file. It would be better to refactor a bit existing styles using `styled-components` first - just because of consistency and maintainability

Reply:
Sounds good I'll move the styles to styled components.